keyboards/annepro2/annepro2.c

```diff
@@ -226,6 +226,60 @@ bool process_record_kb(uint16_t keycode, keyrecord_t *record) {
                 else ap2_led_enable();
                 return true;
             #endif
+            
+            case KC_AP_RGB_VAI:
+				if (record->event.pressed) {
+					if (get_mods() & MOD_MASK_SHIFT) {
+						rgb_matrix_increase_hue();
+						return false;
+					} else if (get_mods() & MOD_MASK_CTRL) {
+						rgb_matrix_decrease_hue();
+						return false;
+					} else {
+						rgb_matrix_increase_val();
+					}
+				}
+                return true;
+                
+            case KC_AP_RGB_VAD:
+				if (record->event.pressed) {
+					if (get_mods() & MOD_MASK_SHIFT) {
+						rgb_matrix_increase_sat();
+						return false;
+					} else if (get_mods() & MOD_MASK_CTRL) {
+						rgb_matrix_decrease_sat();
+						return false;
+					} else {
+						rgb_matrix_decrease_val();
+					}
+				}
+                return true;
+                
+            case KC_AP_RGB_TOG:
+				if (record->event.pressed) {
+					if (get_mods() & MOD_MASK_SHIFT) {
+						rgb_matrix_increase_speed();
+						return false;
+					} else if (get_mods() & MOD_MASK_CTRL) {
+						rgb_matrix_decrease_speed();
+						return false;
+					} else {
+						rgb_matrix_toggle();
```

Comment:
We can't just toggle the RGB matrix, the other HT32 MCU needs to know when the RGB matrix is off to enter low-power mode (specially useful for the ones who need to extend the battery life while using BT)